internal/feature/logodetection/README.md

```
@@ -0,0 +1,497 @@
+```
+
+## API仕様
+
+### POST /v1/logo/detect
+
+画像をアップロードしてロゴを検出します。JWT認証が必要です。
+
+**Content-Type**: `multipart/form-data`
+
+**リクエストフィールド**
+| フィールド | 型 | 必須 | 説明 |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)